gallery/src/data-visualization/ReactVisLineGraphPage.tsx

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import { NxTable, NxTableHead, NxTableRow, NxTableCell, NxTableBody } from '@sonatype/react-shared-components';
+import React from 'react';
+import { GalleryDescriptionTile, GalleryExampleTile } from '../gallery-components/GalleryTiles';
+import ReactVisLineGraphExample from './examples/ReactVisLineGraphExample';
+
+const reactVisLineGraphExampleCode = require('./examples/ReactVisLineGraphExample?raw');
+
+const ReactVisLineGraphPage = () =>
+  <>
+    <GalleryDescriptionTile >
+      <p className="nx-p">
+        Line graphs are part of  <code className="nx-code">{'<LineSeries />'}</code> in React-vis.
+      </p>
+      <p className="nx-p">
+        The following documentation only contains basics on how to get started with creating basic line graphs. <br />
+        For the full, official API documentation, please click {' '}
+        <a href="https://uber.github.io/react-vis/documentation/series-reference/line-series" target="_blank">here.</a>
```

Comment:
Use `NxTextLink`

Reply:
> Use `NxTextLink`

Fixed.
https://github.com/sonatype/sonatype-react-shared-components/pull/442/files#diff-d56acd07c265273046a9edbf323c951b328af933c0b40e157632773f54d08452R23-R28

Reply:
> Fixed.
https://github.com/sonatype/sonatype-react-shared-components/pull/442/files#diff-d56acd07c265273046a9edbf323c951b328af933c0b40e157632773f54d08452R23-R28

Needs the `external` prop